lib/ddtrace/ext/distributed.rb

```diff
@@ -20,8 +20,10 @@ module DistributedTracing
       PROPAGATION_STYLE_DATADOG = 'Datadog'.freeze
       PROPAGATION_STYLE_B3 = 'B3'.freeze
       PROPAGATION_STYLE_B3_SINGLE_HEADER = 'B3 single header'.freeze
-      PROPAGATION_INJECT_STYLE_ENV = 'DD_PROPAGATION_INJECT_STYLE'.freeze
-      PROPAGATION_EXTRACT_STYLE_ENV = 'DD_PROPAGATION_EXTRACT_STYLE'.freeze
+      PROPAGATION_INJECT_STYLE_ENV = 'DD_PROPAGATION_STYLE_INJECT'.freeze
+      PROPAGATION_EXTRACT_STYLE_ENV = 'DD_PROPAGATION_STYLE_EXTRACT'.freeze
+      PROPAGATION_INJECT_STYLE_ENV_OLD = 'DD_PROPAGATION_INJECT_STYLE'.freeze
```

Comment:
Can you write a comment on these old valued about them being erroneously defined by us? This should allow us to confidently remove them in the future.

Reply:
yup will update